.github/workflows/test-and-publish.yaml

```diff
@@ -41,6 +42,11 @@ jobs:
             ./_spidermonkey_install/*
           key: spidermonkey-${{ runner.os }}-${{ runner.arch }}
           lookup-only: true # skip download
+      - name: Setup Poetry
+        if: ${{ steps.cache-spidermonkey.outputs.cache-hit != 'true' }}
+        uses: snok/install-poetry@v1
+        with:
+          version: 1.5.1
```

Comment:
https://github.com/Distributive-Network/PythonMonkey/blob/97944735bb538d0e06f0793dbe0e5b0c08f1618a/setup.sh#L25-L26

Poetry has not been added to PATH after installation, so we need to install and setup PATH beforehand.